x-pack/legacy/plugins/rollup/public/crud_app/sections/job_create/steps/step_review.js

```diff
@@ -122,12 +127,25 @@ export class StepReviewUi extends Component {
         {this.renderTabs()}
 
         <EuiErrorBoundary>
-          <JobDetails
-            job={job}
-            json={json}
-            tab={selectedTab}
-          />
+          <JobDetails job={job} json={json} tab={selectedTab} />
         </EuiErrorBoundary>
+
+        <EuiSpacer size="m" />
+
+        <EuiCheckbox
+          id="rollupJobToggleJobStartAfterCreation"
+          data-test-subj="rollupJobToggleJobStartAfterCreation"
+          checked={startJobAfterCreation}
+          label={
+            <span>
+              <FormattedMessage
+                id="xpack.rollupJobs.create.startJobImmediately"
```

Comment:
this i18n key could be better as `xpack.rollupJobs.create.startJobImmediatelyLabel` (addition of `Label` at end). in general we try to make sure the last word of i18n keys reflect its role in the interface: https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types